tiptapy/extras.py

```diff
@@ -7,15 +7,18 @@ class FeaturedImage(BaseNode):
     def inner_render(self, node) -> str:
         special_attrs_map = {'caption': 'figcaption'}
         attrs = node.get("attrs", {})
+        html = ""
         attrs_s = " ".join(f'{k}="{v}"'
                            for k, v in attrs.items()
                            if k not in special_attrs_map and v.strip()
                            )
-        html = f"<picture><img {attrs_s}></picture>"
-        if attrs.get('caption', '').strip():
-            tag = special_attrs_map['caption']
-            html += f"<{tag}>{attrs['caption']}</{tag}>"
-        return f'<figure class="featured-image">{html}</figure>'
+        if attrs_s:
+            inner_html = f"<img {attrs_s}>"
+            if attrs.get('caption', '').strip():
```

Comment:
In the embed code yesterday for the caption part I did this. But I find the above method to passing all the tests. 

Should I change it here too @gauravr @shon?

```
caption = attrs.get('caption') or ''
if caption.strip():
    tag = special_attrs_map['caption']